update-instructions.md

```diff
@@ -102,6 +102,10 @@ If there is a new Minecraft version:
 
 </details>
 
+### GDLauncher Carbon
+
+To be continued...
```

Comment:
todo

Reply:
This should be fixed within the latest commit (https://github.com/Fabulously-Optimized/wiki/pull/109/commits/7133e69756bd59b992c4d6ca1f8528e4cf9c9a6b). I have added instructions on how to update the modpack.